docs/data/gridftp.md

```diff
@@ -83,22 +141,38 @@ This is usually one XML node spread over multiple lines. Note that comments (\#)
 
 #### Selecting a collector host ####
 
-The collector is the central server which logs the GridFTP transfers into a database. There are usually three options:
+The collector is the central server which logs the GridFTP transfers into a database. There are usually two options:
 
 1. **OSG Transfer Collector**: This is the primary collector for transfers in the OSG. Use `CollectorHost="gratia-osg-prod.opensciencegrid.org:80"`.
```

Comment:
@matyasselmeci this guide is full of manual Gratia probe configuration. Does `osg-configure` handle GridFTP probe config?